### DISCUSSION TITLE
right click on taskbar no longer brings up EP, I'm getting StartAllBack instead

### DISCUSSION THREAD

**Discussion by kwagraphics** [valinet/ExplorerPatcher#3602] [Q&A]
This has been going on for quite a long time. When I right click on taskbar it no longer brings up EP, I'm getting StartAllBack instead. I was hoping some update would fix it, but no joy so far.

I got an update for SAB (3.8.6) and that reminded me about EP not working right.

EP version is 22621.3527.65.5
Windows 11 OS Build 22631.4037

I have a short cut for EP so I do have easy access to the program. Still, something isn't right.

I looked to see if there were any postings about this and found a couple of issues with right click not working properly, but not exactly what I'm experiencing. I changed the "Do not change the taskbar context menu" setting and it didn't make a difference. And my taskbar is on the bottom.

I had another piece of software I thought could be causing a conflict and uninstalled it. And I rebooted after doing that.

**Comment by pyrates999** [comment]
Install the pre-release here: https://github.com/valinet/ExplorerPatcher/releases

**Reply by kwagraphics** [reply]
I installed the pre-release 22621.3880.66.4. I rebooted. Still having the same problem. And now I have a white border around the system tray.

It's not the end of the world. I just hate it when I know something isn't right and I can't fix it. Makes me wonder if there are other gremlins in the works!

I hate to do it, but I'll try uninstalling SAB. See what happens and then reinstall it.

I'm not feeling good and when I don't feel good things usually go to pot when I'm doing this kind of stuff. I've already had a bad experience tonight doing something simple (changing HBO/Max's streaming provider). It did not go well!

**Reply by pyrates999** [reply]
what taskbar did you set EP to use?

**Reply by kwagraphics** [reply]
Windows 10.

**Reply by pyrates999** [reply]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

**Reply by kwagraphics** [reply]
Did that, rebooted, no change. I even gave Windows 11 a go and the right click still brings up SAB.

I did the uninstall of SAB. And since I'm not feeling good I wish I could tell you what happened when I right clicked and selected properties. I'm sure it didn't bring up EP, but I can't for the life of me remember what properties I did get. 

My start menu didn't change. I did a backup of it just in case. I didn't want to reinvent the wheel. I'm not feeling up to doing that. The taskbar was an ugly hot mess I remember that!

I feel bad putting you through all this.

**Reply by pyrates999** [reply]
no worries

**Comment by milestprower92** [comment]
> This has been going on for quite a long time. When I right click on taskbar it no longer brings up EP, I'm getting StartAllBack instead. I was hoping some update would fix it, but no joy so far.
> 
> I got an update for SAB (3.8.6) and that reminded me about EP not working right.
> 
> EP version is 22621.3527.65.5 Windows 11 OS Build 22631.4037
> 
> I have a short cut for EP so I do have easy access to the program. Still, something isn't right.
> 
> I looked to see if there were any postings about this and found a couple of issues with right click not working properly, but not exactly what I'm experiencing. I changed the "Do not change the taskbar context menu" setting and it didn't make a difference. And my taskbar is on the bottom.
> 
> I had another piece of software I thought could be causing a conflict and uninstalled it. And I rebooted after doing that.

ExplorerPatcher and StartAllBack should not be used together
